spec/jwt_spec.rb

```diff
@@ -438,6 +438,13 @@
     end
   end
 
+  context 'when token is not a String' do
+    it 'raises JWT::DecodeError' do
+      expect { JWT.decode(nil, nil, true) }.to raise_error(JWT::DecodeError, 'NilClass JSON web token')
+      expect { JWT.decode(1, nil, true) }.to raise_error(JWT::DecodeError, 'Integer JSON web token')
```

Comment:
Integer is still Fixnum in some older rubies that the gem supports (See the failing tests)